main.tex

```diff
@@ -59,23 +59,29 @@
 %
-For each variant we predict a resistance phenotype based on genetic evidence for positive selection, patterns of linkage between variants, and functional evidence from other species.
+For each novel variant allele we predict a putative resistance phenotype based on genetic evidence for positive selection, patterns of genetic linkage between variants, location of the variant within the protein domain architecture, and functional evidence from other species.
 %
-We then analyse the genetic backgrounds on which resistance variants are found, to refine our understanding of the origins and spread of resistance between species and geographical locations.
+Thirteen of these novel alleles occur almost exclusively on haplotypes carrying the known L995F resistance allele, and appear to be secondary mutations that may enhance or compensate for the primary resistance phenotype.
 %
-Using networks and hierarchical clustering methods we identify ten distinct haplotype clusters caused by selective sweeps at this resistance locus, of which five appear to be localised to a single geographical location, and five have spread between two or more countries.
+We also report an I1527T substitution which is adjacent to a predicted pyrethroid binding site in the channel molecule, and occurs in tight linkage with either of two alleles causing a V402L substitution, similar to a combination of substitutions recently found to cause pyrethroid resistance in Aedes @@TODO.
 %
-The most successful and widespread haplotype cluster (F1) originates in West Africa and has subsequently spread to countries in Central and Southern Africa.
+We then analyse the genetic backgrounds on which known and putative resistance alleles are found, to determine which alleles have experienced recent positive selection, and to refine our understanding of the origins and spread of resistance between species and geographical locations.
 %
-Our results demonstrate that the molecular basis of pyrethroid resistance in African malaria vectors is much more complex than previously appreciated, and provide a foundation for the design of new genetic tools to track the spread insecticide resistance and to inform vector control.
+We identify ten distinct haplotype clusters under selection, five of which carry the L995F resistance allele, four of which carry L995S, and one of which carries I1527T.
+%
+Five of these clusters are localised to a single geographical location, and five comprise haplotypes from two or more countries, indicating the geographical spread of resistance.
+%
+We also find evidence for multiple introgression events transmitting resistance alleles between the two mosquito species.
+%
+Our results demonstrate that the molecular basis of pyrethroid resistance in African malaria vectors is more complex than previously appreciated, and provide a foundation for the design of new genetic tools to track the spread insecticide resistance and to inform vector control.
 %%
```

Comment:
I've made some edits to the abstract to try and increase the readability a little, and also add some specific results to make the main findings more concrete and apparent. This does increase the abstract length but we're still under 420 words which is fine for some journals. And I figured better to be a bit longer now and lay it all out, we can trim down later if needed.

Reply:
Looks good.